schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+id INT,
+name TEXT,
+date_of_birth DATE,
+escape_attempts INT,
+neutered BIT,
```

Comment:
- [ ] As mentioned in the project requirements, this should be a boolean. It should have either `true` or `false` as their values and not `0` or `1` :+1: